### ISSUE TITLE
Converted result of markdown yaml header include local path

### ISSUE THREAD

**Issue by isoppp** [gridsome/gridsome#87]
## Description

Field included like extension string include local path

### Steps to reproduce

Code to reproduce:
https://github.com/isoppp/issue-examples/tree/master/gridsome/markdown-convert-problem

### Expected result

in Markdown:

```
---
description: 'Nuxt.js'
tags:
    - 'Nuxt.js'
---
```

I expected description and tags output Nuxt.js

### Actual result

GraphQL output:

```
  "data": {
    "allBlogPost": {
      "edges": [
        {
          "node": {
            "description": "/Users/isoppp/personal/issue-examples/gridsome/markdown-convert-problem/blog/Nuxt.js",
            "tags": [
              "/Users/isoppp/personal/issue-examples/gridsome/markdown-convert-problem/blog/Nuxt.js"
            ],
          }
        }
      ]
    }
  }
```

**Comment by isoppp** [comment]
Thanks reply!
I'm temporarily avoiding it by the following way.

---

(It's note for other same people)
Add dummy text (e.g `$`) to markdown then replace them in Vue.

```
---
description: 'xxx.js$'
---
```

replace in vue:

```
description.replace(/\$$/, '')
```

**Comment by isoppp** [comment]
Closing, I confirmed that this issue was resolved on v 0.5.0.
